src/renderer/bootstrap.tsx

```diff
@@ -19,6 +20,17 @@ import { App } from "./components/app";
 import { LensApp } from "./lens-app";
 import { themeStore } from "./theme.store";
 
+/**
+ * If this is a development buid, wait a second to attach
+ * Chrome Debugger to renderer process
+ * https://stackoverflow.com/questions/52844870/debugging-electron-renderer-process-with-vscode
+ */
+async function attachChromeDebugger() {
+  if (remote.process.defaultApp) {
+    await new Promise(r => setTimeout(r, 1000));
```

Comment:
[nit] prefer the `resolve` instead of just `r` (can be confusing)
```suggestion
    await new Promise(resolve => setTimeout(resolve, 1000));
```